link_palindrome.c

```diff
@@ -0,0 +1,57 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <assert.h>
+#include <stdbool.h>
+#include "link_palindrome.h"
+
+void Check_Input(List* head)
+{
+	assert(head != NULL);
+	return;
+}
+
+int Get_Link_List_Length(List* head)
+{
+	int length = 0; 
+	List* temp = head; 
+
+	while(temp != NULL)
+	{
+		length++;
+		temp = temp -> next;
+	}
+
+	return(length);
+}
+
+bool Check_Palindrome(List* head, int length)
+{
+	int even_or_odd_length = length % 2;
```

Comment:
You only ever use this to check if the length of the list is odd, so name the variable as such. It is also representing an "either or" state, so something like "bool isOddLength = (length % 2 != 0);" would be more clear